cmd/buck/cli/cli.go

```diff
@@ -56,6 +56,7 @@ func Init(baseCmd *cobra.Command) {
 	initCmd.Flags().BoolP("private", "p", false, "Obfuscates files and folders with encryption")
 	initCmd.Flags().String("cid", "", "Bootstrap the bucket with a UnixFS Cid from the IPFS network")
 	initCmd.Flags().BoolP("existing", "e", false, "Initializes from an existing remote bucket if true")
+	initCmd.Flags().Bool("sync", false, "Syncs local state with remote, i.e., discards local changes if true")
```

Comment:
not a blocker by any means, but it seems like "overwrite" or "force" would be more common terms to use here? but I might be missing some nuance to what sync really means in this case.

Reply:
I agree with you there. I tend to "force"

Reply:
We use `force` with `pull` in a way that means, "force pull all files, even if they exist locally", which is different that the meaning here... so, I opted to avoid the confusion. However! I have removed the `sync` flag in favor of two other ones that cover more use cases. See PR description.